homeassistant/components/atome/sensor.py

```diff
@@ -248,5 +279,6 @@ def update(self):
         else:
             self._attr_state = getattr(self._data, f"{self._sensor_type}_usage")
             self._attr_extra_state_attributes = {
-                "price": getattr(self._data, f"{self._sensor_type}_price")
+                "price": getattr(self._data, f"{self._sensor_type}_price"),
+                "last_reset": dt_util.parse_datetime(getattr(self._data, f"{self._sensor_type}_last_reset"))
```

Comment:
Please set `last_reset` with the corresponding Entity property or class/instance attribute. Ie, it should not be part of extra attributes. Note that only accumulating measurements like energy kWh should set `last_reset`.

https://developers.home-assistant.io/docs/core/entity/sensor#properties

Reply:
Thanks for pointing to the documentation. I am fairly new to Python and this is my first attempt at HomeAssistant code fixing.

Reply:
Done